docs/examples/Reinforcement_Learning_Sensor_Manager.py

```diff
@@ -0,0 +1,951 @@
+
+yps = range(0, 100, 10)  # y value for prior state
+truths = []
+ntruths = 3  # number of ground truths in simulation
+time_max = 50  # timestamps the simulation is observed over
+
+xdirection = 1
+ydirection = 1
+
+# Generate ground truths
+for j in range(0, ntruths):
+    truth = GroundTruthPath([GroundTruthState([0, xdirection, yps[j], ydirection],
+                                              timestamp=start_time)],
+                            id=f"id{j}")
+
+    for k in range(1, time_max):
+        truth.append(
+            GroundTruthState(transition_model.function(truth[k - 1], noise=True, time_interval=timedelta(seconds=1)),
+                             timestamp=start_time + timedelta(seconds=k)))
+    truths.append(truth)
+
+    # alternate directions when initiating tracks
+    xdirection *= -1
+    if j % 2 == 0:
+        ydirection *= -1
+
+# %%
+# Plot the ground truths. This is done using the `~.Plotterly` class from Stone Soup.
+
+from stonesoup.plotter import Plotterly
+
+# Stonesoup plotter requires sets not lists
+truths_set = set(truths)
+
+plotter = Plotterly()
+plotter.plot_ground_truths(truths_set, [0, 2])
+plotter.fig
+
+# %%
+# Create sensors
+# --------------
+# Create a sensor for each sensor management algorithm. This tutorial uses the
+# `~.RadarRotatingBearingRange` sensor. This sensor is an `~.Actionable` so
+# is capable of returning the actions it can possibly
+# take at a given time step and can also be given an action to take before taking
+# measurements.
+# See the Creating an Actionable Sensor Example for a more detailed explanation of actionable sensors.
+# 
+# The `~.RadarRotatingBearingRange` has a dwell centre which is an `~.ActionableProperty`
+# so in this case the action is changing the dwell centre to point in a specific direction.
+# 
+
+from stonesoup.types.state import StateVector
+from stonesoup.sensor.radar.radar import RadarRotatingBearingRange
+
+sensorA = RadarRotatingBearingRange(
+    position_mapping=(0, 2),
+    noise_covar=np.array([[np.radians(0.5) ** 2, 0],
+                          [0, 1 ** 2]]),
+    ndim_state=4,
+    position=np.array([[10], [0]]),
+    rpm=60,
+    fov_angle=np.radians(45),
+    dwell_centre=StateVector([0.0]),
+    max_range=np.inf
+)
+sensorA.timestamp = start_time
+
+sensorB = RadarRotatingBearingRange(
+    position_mapping=(0, 2),
+    noise_covar=np.array([[np.radians(0.5) ** 2, 0],
+                          [0, 1 ** 2]]),
+    ndim_state=4,
+    position=np.array([[10], [0]]),
+    rpm=60,
+    fov_angle=np.radians(45),
+    dwell_centre=StateVector([0.0]),
+    max_range=np.inf
+)
+sensorB.timestamp = start_time
+
+# %%
+# Create the Kalman predictor and updater
+# ---------------------------------------
+# Construct a predictor and updater using the `~.KalmanPredictor` and `~.ExtendedKalmanUpdater`
+# components from Stone Soup. The `~.ExtendedKalmanUpdater` is used because it can be used for both linear
```

Comment:
```suggestion
# Construct a predictor and updater using the :class:`~.KalmanPredictor` and :class:`~.ExtendedKalmanUpdater`
# components from Stone Soup. The :class:`~.ExtendedKalmanUpdater` is used because it can be used for both linear
```